apps/frontend/src/components/fap/Proposals/FapProposalsAndAssignmentsTable.tsx

```diff
@@ -765,6 +771,8 @@ const FapProposalsAndAssignmentsTable = ({
           options={{
             search: true,
             selection: true,
+            pageSize: Math.min(10, maxPageLength),
```

Comment:
Seems odd to show 5 as an option for page size if we're defaulting to the lower of everything or 10

Reply:
The 5 comes from the tables comments default options which as used across the application: https://github.com/material-table-core/core/blob/6303bd78399666e129a91ed28154ddd4d38c22c2/src/defaults/props.options.js#L26 and the 10 is what we set as default before. Should I remove the 5 in this case?